src/k8s/api/v1/cluster.go

```diff
@@ -8,6 +8,13 @@ type GetClusterStatusResponse struct {
 	ClusterStatus ClusterStatus `json:"status"`
 }
 
+// PostClusterBootstrapRequest is used to bootstrap the cluster.
+type PostClusterBootstrapRequest struct {
```

Comment:
Our naming has become a bit weird here.
We initially had "<method><endpoint-name>" for the API side function. 
So, it would be consistent to name this `PostClusterRequest` but that is even more weird. 
Well, the API will likely change several times anyway so we got some time to think about it...

Reply:
I agree, my thought here was since we are also doing `GetClusterStatusRequest` instead of `GetClusterRequest` it might be a bit better in conveying what it is used for.